src/network/citrix/netscaler/snmp/mode/health.pm

```diff
@@ -69,7 +69,7 @@ sub new {
     my $self = $class->SUPER::new(package => __PACKAGE__, %options);
     bless $self, $class;
 
-    $options{options}->add_options(arguments => {});
+    $options{options}->add_options(arguments => { "psu-new-oid:s" => { name => 'psu_new_oid', default => '' }});
```

Comment:
A couple of remarks:
- The new option's name is not very clear to me. It does not introduce a new OID but a new way to translate it. `--alternative-status-mapping` or `--new-status-mapping` or `--old-status-mapping` would be better. Unless the old mapping is still widely used, I would change the default behaviour to the new one, so that recent devices are correctly monitored out-of-the-box, so I would pick `--old-status-mapping`.
- This option should be a boolean flag (ie. `"alternative-status-mapping:b"`), so it will be either undef or 1 (not sure, but defined for sure).